rosbag2_bag_v2_plugins/test/rosbag2_bag_v2_plugins/test_rosbag2_play_rosbag_v2_end_to_end.cpp

```diff
@@ -63,14 +63,16 @@ class PlayEndToEndTestFixture : public Test
 };
 
 TEST_F(PlayEndToEndTestFixture, play_all) {
-  rosbag2_transport::Rosbag2Transport rosbag2_transport;
   rosbag2_storage::StorageOptions storage_options;
   storage_options.uri = database_path_ + "/test_bag_end_to_end.bag";
   printf("storage_options uri %s\n", storage_options.uri.c_str());
   storage_options.storage_id = "rosbag_v2";
   rosbag2_transport::PlayOptions play_options;
   play_options.read_ahead_queue_size = 1000;
-  rosbag2_transport.play(storage_options, play_options);
+
+  rosbag2_transport::Player player(
+    std::make_unique<rosbag2_cpp::Reader>(), storage_options, play_options);
```

Comment:
this line makes me think we should probably have another constructor overload in the player API

Reply:
With default Reader - and only takes storage/play options?

Reply:
done in d7cb529